README.md

```diff
@@ -0,0 +1,53 @@
+# Welcome to cascade 👋
+[![Documentation](https://img.shields.io/badge/documentation-yes-brightgreen.svg)](https://github.com/the-dr-lazy/cascade/wiki)
+[![Twitter: the\_dr\_lazy\_](https://img.shields.io/twitter/follow/the\_dr\_lazy\_.svg?style=social)](https://twitter.com/the\_dr\_lazy\_)
+https://github.com/the-dr-lazy/cascade/blob/master/LICENSE
+> A project management service just to show that real-world Haskell is awesome.
+
+### 🏠 [Homepage](https://github.com/the-dr-lazy/cascade)
+
+### ✨ [Demo](www.demo.com)
+
+## Install
+
+```sh
+install command
+```
+
+## Usage
+
+```sh
+how to use it
+```
```

Comment:
We really don't know what is gonna happen to the design of Cascade. So, until then let's remove this section for now.